raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -753,11 +751,13 @@ void start(int nodeId) {
                 FETCH_TIMEOUT_MS, persistentState.store, time, logContext, random);
             Metrics metrics = new Metrics(time);
 
-            Map<Integer, InetSocketAddress> voterConnectionMap = voters.stream()
-                .collect(Collectors.toMap(
-                    Function.identity(),
-                    this::nodeAddress
-                ));
+            StringBuilder votersString = new StringBuilder();
```

Comment:
nit: since we have the same code in a couple places, maybe we can factor it into a `RaftTestUtil`?